frame/support/procedural/src/pallet/expand/call.rs

```diff
@@ -137,6 +137,8 @@ pub fn expand_call(def: &mut Def) -> proc_macro2::TokenStream {
 	let count = COUNTER.with(|counter| counter.borrow_mut().inc());
 	let macro_ident = syn::Ident::new(&format!("__is_call_part_defined_{}", count), span);
 
+	let capture_docs = if cfg!(feature = "no-metadata-docs") { "never" } else { "always" };
```

Comment:
would have liked a test that would fail if the rust team decided to change these two pairs to `Never` and `Always`, but I guess that's extremely unlikely at this point.